### PR TITLE
Remove Log::event

### PR DESCRIPTION
**Pull request by Jellyfrog**
Use the Eventlog class directly instead

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Blocked by https://github.com/librenms/librenms/pull/14546

File: tests/OSModulesTest.php [hunk 1]
```diff
@@ -165,13 +165,6 @@ public function dumpedDataProvider()
 
     private function stubClasses(): void
     {
-        $this->app->bind('log', function ($app) {
-            $mock = \Mockery::mock('\App\Facades\LogManager[event]', [$app]);
-            $mock->shouldReceive('event');
-
-            return $mock;
-        });
-
```
Comment (Jellyfrog) [inline tests/OSModulesTest.php:174]: @murrant what's left here is this, I suppose we could stub it out in the same way as astext?

Comment (murrant) [reply tests/OSModulesTest.php:174]: You can't mock static methods.

File: app/Models/Eventlog.php [hunk 2]
```diff
@@ -38,17 +38,33 @@ class Eventlog extends DeviceRelatedModel
...
+     * Log events to the event table
+     *
+     * @param  string  $text
+     * @param  Device|int|null  $device
+     * @param  string  $type
+     * @param  int  $severity
+     * @param  int|string|null  $reference
      */
-    public static function log($text, $device = null, $type = null, $severity = Alert::INFO, $reference = null)
+    public static function _log($text, $device = null, $type = null, $severity = Alert::INFO, $reference = null): void
```
Comment (murrant) [inline app/Models/Eventlog.php:67]: This should be non-static

File: app/Models/Eventlog.php [hunk 3]
```diff
@@ -59,6 +75,10 @@ public static function log($text, $device = null, $type = null, $severity = Aler
             'username'  => (class_exists('\Auth') && Auth::check()) ? Auth::user()->username : '',
         ]);
 
+        if (is_numeric($device)) {
```
Comment (murrant) [inline app/Models/Eventlog.php:78]: Hmm, I think I purposely required a Device model.  I let the event_log() function convert int to Device before calling Log::event()

Comment (Jellyfrog) [reply app/Models/Eventlog.php:78]: log_event probably does

**Review by murrant — APPROVED** [review]

**Pull request merged**